Router.svelte

```diff
@@ -177,6 +198,12 @@ export function link(node) {
 
     // Add # to every href attribute
     node.setAttribute('href', '#' + href)
+
+    // use onclick to push the route, so we can check if browser back button is clicked
+    node.onclick = () => {
```

Comment:
The first releases of svelte-spa-router used an onclick event, but we had to remove that (and replace it with the current code you see here) because of #33 .

I can't think right now of a better way to refactor this code so it doesn't cause issues with the case in #33. It might just be something we have to live with, but I felt like bringing this up to you so you're at least aware of it. If you have ideas on how to do this without onclick, feel free to refactor!

Reply:
This should work as expected (quick test it with chrome). Cause the setAttribute # is still there.
But i will test it with different browsers.